### ISSUE TITLE
Incorrect typing for values on EnumType.__init__()

### ISSUE THREAD

**Issue by markedwards** [mirumee/ariadne#1039]
Ariadne 0.18.0 has updated the `EnumType` class in a way that causes the `values` argument of `__init__()` to become typed, but the typing is incorrect:

```
    def __init__(
        self, name: str, values: Union[Dict[str, Any], enum.Enum, enum.IntEnum]
    ) -> None:
```

should be:

```
    def __init__(
        self, name: str, values: Union[Dict[str, Any], Type[enum.Enum], Type[enum.IntEnum]]
    ) -> None:
```

**Comment by rafalp** [comment]
Hey, thanks. Pull request is welcome

**Comment by markedwards** [comment]
Done

**Comment by markedwards** [comment]
Thanks for the quick accept.  Any rough idea when this might get released?  Until it does, 0.18.0 will cause tons of mypy explosions for us.  I can always pin, but I figured I'd ask about ETA.

**Comment by rafalp** [comment]
Tomorrow. Its almost 8pm here ;)
